README.md

```diff
@@ -29,15 +29,24 @@ This site relies heavily on node and npm.
 1. Make sure you are using at least node v.5+ and latest npm; 
 if not install [nvm](https://github.com/creationix/nvm) to get node going on your machine.
 
-1. Install npm `gulp` package *globally*: `npm install -g gulp`
+1. Install global npm packages by running `./scripts/before-install.sh`
```

Comment:
Can you revert this line? Not all windows users use a console that can run bash scripts.